### PR TITLE
Wrap settings at 140 columns

### PR DESCRIPTION
**Pull request by nik9000**
and remove the checkstyle suppressions.

### CODE REVIEW

File: core/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java [hunk 1]
```diff
@@ -143,15 +143,17 @@
             Map<String, Settings> groups = s.getAsGroups();
             for (String key : SimilarityService.BUILT_IN.keySet()) {
                 if (groups.containsKey(key)) {
-                    throw new IllegalArgumentException("illegal value for [index.similarity."+ key + "] cannot redefine built-in similarity");
-                }
+                        throw new IllegalArgumentException(
+                                "illegal value for [index.similarity." + key + "] cannot redefine built-in similarity");
+                    }
```
Comment (dakrone) [inline core/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java]: This indentation is incorrect

**Comment by dakrone** [comment]
Left one comment, could probably bikeshed this to oblivion since everyone has their favorite multi-line indentation style, but LGTM

File: core/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java (hunk 1 above)
Comment (nik9000) [inline core/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java]: Good old Eclipse, failing to notice the world around it. I, also, failed to notice. But I pushed a fix!

**Comment by nik9000** [comment]
> favorite multi-line indentation style

All of them are better then never wrapping the line at all. I'd love to hear about your favorites sometime because I don't really have a favorite.

**Comment by nik9000** [comment]
Thanks for catching my mistake @dakrone !

**Pull request merged**